### DISCUSSION TITLE
태블랫, 모바일 화면에서는 게시물의 width가 동일하지만 pc 화면에서는 상이합니다.

### DISCUSSION THREAD

**Discussion by JungJaeLee-JJ** [pronist/hELLO#85] [Q&A] [answered]
오늘 스킨을 변경한 저의 블로그 입니다. https://blog.jjlovechoco.com/7

pc로 링크에 접속해보시면, 사진의 넓이와 다른 텍스트들의 넓이가 다른 것을 볼 수 있습니다. 어떻게 해결할 수 있을까요?

**Comment by pronist** [comment, marked as answer]
@JungJaeLee-JJ 사진의 넓이와 텍스트의 넓이가 다른 것은 의도한 것입니다. 조금 더 표현을 다채롭게 하기 위한 장치인 것이지요. 만약 본문 폭에 맞추고 싶다면 에디터에 중앙정렬이 아닌 폭맞춤과 같은 다른 옵션을 주셔야 할겁니다.

**Comment by JungJaeLee-JJ** [comment]
폭맞춤을 했더니 해결되었습니다! 감사합니다.
